### ISSUE TITLE
can not run on clusters created not by CF stack - for example kops

### ISSUE THREAD

**Issue by szuecs** [zalando-incubator/kube-ingress-aws-controller#95]
https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/master/aws/ec2.go#L227-L255
makes string assumption on aws internal tags to find a security group which are only possible to be created by a CF stack.

Kops does not use CF stack to create a cluster so it's not possible to run this controller in a kops cluster

**Comment by mikkeloscar** [comment]
Related to #77

**Comment by arun-gupta** [comment]
Has an issue been logged in kops repo?

**Comment by hjacobs** [comment]
@arun-gupta no, there is no issue in the kops repo AFAIK, but I think not using CF should not be a problem. The ingress controller should work when not using Cloud Formation, too.

**Comment by mikkeloscar** [comment]
Fixed in #96

**Comment by szuecs** [comment]
So one problem with kops is that they don't provision everything with the right tags, for example SecurityGroup `api-elb.<cluster-name>` has not the tag:

    kubernetes.io/cluster/<cluster-id>=owned

Additionally we need the tag:

    kubernetes:application=kube-ingress-aws-controller
